src/utils.ts

```diff
@@ -46,7 +46,7 @@ export function handleErrors(func: () => Promise<void>) {
         try {
             await func();
         }
-        catch (err) {
+        catch (err: any) {
```

Comment:
This seems like a drive-by typing change.

Reply:
looking at MR now, it should have been two MRs with the a "assign a scheme" depending on the "update packages". I can fix it if needed - I was kinda in a hurry to compile a working version for myself.

Reply:
I kept the change, but I think the `any` is no longer necessary after merge